project/Dependencies.scala

```diff
@@ -25,6 +27,13 @@ object Dependencies {
     val r2dbcSpi = "io.r2dbc" % "r2dbc-spi" % "1.0.0.RELEASE" // ApacheV2
     val r2dbcPool = "io.r2dbc" % "r2dbc-pool" % "1.0.0.RELEASE" // ApacheV2
     val r2dbcPostgres = "org.postgresql" % "r2dbc-postgresql" % "1.0.0.RELEASE" // ApacheV2
+
+    // bump Netty dependency due to CVE-2022-41915
+    // FIXME remove this when r2dbc-postgresql includes a later Netty version
+    val nettyHandlerProxy = "io.netty" % "netty-handler-proxy" % NettyVersion
+    val nettyResolverDns = "io.netty" % "netty-resolver-dns" % NettyVersion
+    val nettyResolverDnsMacos = "io.netty" % "netty-resolver-dns-native-macos" % NettyVersion classifier "osx-x86_64" classifier "osx-aarch_64"
```

Comment:
I had problem with this first, but I had not used the right artifact name. Seems like the build is not failing when using classifier. Anyway, this works, and generates pom:
```
<dependency>
            <groupId>io.netty</groupId>
            <artifactId>netty-resolver-dns-native-macos</artifactId>
            <version>4.1.89.Final</version>
            <classifier>osx-aarch_64</classifier>
        </dependency>
        <dependency>
            <groupId>io.netty</groupId>
            <artifactId>netty-resolver-dns-native-macos</artifactId>
            <version>4.1.89.Final</version>
            <classifier>osx-x86_64</classifier>
        </dependency>
```

Reply:
I ran tests on m1 against Postgres, seems to work fine! 👍

Reply:
thanks for verifying that